<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interfata</class>
 <widget class="QWidget" name="interfata">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="chatTab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>871</width>
     <height>461</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="USER">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
rgb(241, 241, 241)
}</string>
    </property>
    <attribute name="title">
     <string>users</string>
    </attribute>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>411</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="UsersBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>621</width>
       <height>431</height>
      </rect>
     </property>
     <property name="title">
      <string>Users</string>
     </property>
     <widget class="QListWidget" name="chat">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>461</width>
        <height>341</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget{
rgb(241, 241, 241)
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mesajLine">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>370</y>
        <width>461</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sendButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>380</y>
        <width>71</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>send</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="groups">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
rgb(238, 238, 238)
}</string>
    </property>
    <attribute name="title">
     <string>groups</string>
    </attribute>
    <widget class="QListWidget" name="GroupNames">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>701</width>
       <height>421</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QListWidget" name="chatGroup">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>411</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="groupLine">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>380</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="MembersNames">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>161</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="groupSend">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>380</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>200</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>AddMembers</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="CreateGroup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>CreateGroup</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="LogoutButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>10</y>
     <width>90</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
